contrib/charts/navigator/templates/apiserver-pvc.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.apiserver.persistence.enabled }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}-apiserver
+  annotations:
+  {{- if .Values.apiserver.persistence.storageClass }}
+    volume.beta.kubernetes.io/storage-class: {{ .Values.apiserver.persistence.storageClass | quote }}
+  {{- else }}
+    volume.alpha.kubernetes.io/storage-class: default
```

Comment:
Why `alpha` and `beta` in these two branches?

Reply:
Ah, I see that the "default" storage class name is an alpha feature.
Need to fix that in the cassandra chart then.